cpp/Tooling.md

```diff
@@ -2,6 +2,18 @@
 
 ## C++ Compiler
 
+### C++ Standard
+
+The C++ standard officially supported at BBP is C++14.
+An source project can support C++11 if this is a requirement for its community.
```

Comment:
typo?

Reply:
yes thank you. Do not hesitate to provide _suggestions_ that can be committed right-away.